csharp-basics/exercises/Scooters/Scooters/RentalCompany.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections;
+
+namespace Scooters
+{
+    public partial class RentalCompany : IRentalCompany
+    {
+        private string _name;
+        public IScooterService _scooterService = new ScooterService();
+        List<RentalData> _rentalHistory = new List<RentalData>();        
+
+        public RentalCompany(string CompanyName)
+        {
+            _name = CompanyName;
+        }
+
+        public RentalCompany(string CompanyName, ScooterService scooterService, List<RentalData> rentalHistory)
+        {
+            _name = CompanyName;
+            _scooterService = scooterService;
+            _rentalHistory = rentalHistory;
+        }
+
+        public string Name()
```

Comment:
Šis bus vienkārši property. :)

Reply:
Nomainīts

Reply:
Bija cita _companyB, jo gribēju pārbaudīt konstruktoru ar vienu argumentu. Tā kā Name tagad ir property, un Interface tas var būt tiaki static, 14 tests pārtaisīts uz to pašu companyA.